packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -36,9 +50,28 @@ const Toolbar = ({
     customProps.className,
   );
 
+  const additionalToolbarProps = {};
+
+  if (ariaLabel) {
+    additionalToolbarProps['aria-label'] = ariaLabel;
+  }
+
+  if (ariaLabelledBy) {
+    additionalToolbarProps['aria-labelledby'] = ariaLabelledBy;
+  }
+
   const items = React.Children.map(children, item => (item ? <div className={cx('item')}>{item}</div> : item));
 
-  return (<div {...customProps} className={toolbarClassNames} role="toolbar">{items}</div>);
+  return (
+    <div
+      {...customProps}
+      {...additionalToolbarProps}
```

Comment:
> "Will be used over ariaLabel if both are present."

This approach does not use ariaLabelledBy over ariaLabel if both are provided as the prop description suggests, but instead spreads them both. 

Unless you were meaning that a screenreader may choose to read the 'aria-labelledby' over the 'aria-label', which may or may not be consistently true: 
https://www.davidmacd.com/blog/does-aria-label-override-static-text.html

```suggestion      
      aria-label={!ariaLabelledBy ? ariaLabel : undefined}
      aria-labelledby={ariaLabelledBy}
```

\
\
Also updating or adding new WDIO tests, and using the props on the doc examples showing best practice usage recommendations, would help to be able to test them with screenreaders.

Reply:
Resolved: https://github.com/cerner/terra-core/pull/3678/commits/fe5ebc7bfe746948753b4b606b5b9aa4e2088250